src/Modal/Modal.js

```diff
@@ -120,7 +120,12 @@ export function Modal({ children, title, animationDuration, showClose, onClose,
             <FocusOn onEscapeKey={handleClose} enabled={isOpen}>
               <ModalContainer ref={modalRef}>
                 <Backdrop transitionState={state} onClick={handleBackdropClick} />
-                <ModalContent transitionState={state} onClick={handleContentClick} aria-modal="true" {...props}>
+                <ModalContent
+                  transitionState={state}
+                  onClick={handleContentClick}
+                  aria-modal="true"
+                  tabIndex={!autofocus && 0}
```

Comment:
I'm not sure we need this. The input in the Modal stories still autofocuses without setting to true.

Reply:
That is because the Input itself has an auto focus prop.
https://github.com/heydoctor/molekule/blob/master/src/Modal/Modal.example.js#L39